packages/platform-android/native_modules.gradle

```diff
@@ -487,6 +532,7 @@ ext.applyNativeModulesAppBuildGradle = { Project project, String root = null ->
   task generateNewArchitectureFiles {
     doLast {
       autoModules.generateAndroidMkFile(generatedJniDir, "Android-rncli.mk", androidMkTemplate)
+      autoModules.generateCmakeFile(generatedJniDir, "Android-autolinked.cmake", cmakeTemplate) 
```

Comment:
can we rename this to `Android-rncli.cmake` to keep things fairly consistent?

Reply:
Yup updated 👍